ipm_service/setup.py

```diff
@@ -0,0 +1,31 @@
+import glob
+import os
+
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = 'ipm_service'
+
+setup(
+    name=package_name,
+    packages=find_packages(exclude=['test']),
```

Comment:
I didn't know about this but I'm seeing this in a lot (if not all?) core ROS2 packages. Does this exclude the tests from being installed?

Reply:
Probably, I am also only using a template that I found in some guide or core package :D

Reply:
Do you not generate your python package files using ``pkg create`` (ie.
``ros2 pkg create soccer_ipm --build-type ament_python``)?

Reply:
Normally honestly not, because for most packages we have bit-bots stuff like docs that need to be added either way. And when we started I didn't know this.